package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "weather-app-js",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch"
```

Comment:
Its good practice to add a `dev` and a `build` script. So in each one, you can specify in which mode you want to Webpack run. Something like:
`"dev": "webpack --mode development"`
`"build": "webpack --mode production"`

You can also delete the test script here since it's not being used.